### PR TITLE
[OSCI] Add new osci author bigsamu

### PR DESCRIPTION
**Pull request by BigSamu**
### Description
Add Samuel Valdes Gutierrez (bigsamu) as OSCI author to the repository
  
### Issues Resolved
N/A

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _authors/bigsamu.markdown [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+short_name: bigsamu
+name: Samuel Valdes Gutierrez
+github: bigsamu
+photo: "/assets/media/authors/bigsamu.jpg"
```
Comment (cwperks) [inline _authors/bigsamu.markdown:5]: Please remove the added image and this line.

In the last line of the file with an introduction, please start the line with your name bolded like `**<Name>**`

**Review by cwperks — CHANGES_REQUESTED** [review]
Thank you @BigSamu! Please remove the added image and add your name in the last line with your introduction.

File: _authors/bigsamu.markdown (hunk 1 above)
Comment (BigSamu) [reply _authors/bigsamu.markdown]: Hi @cwperks. I just made the changes! Thanks for the message!

**Comment by joshuarrrr** [comment]
@BigSamu Thanks for the fix - now you just need to fix DCO on all commits following the instructions on https://github.com/opensearch-project/project-website/pull/2085/checks?check_run_id=17062276982

**Comment by BigSamu** [comment]
> @BigSamu Thanks for the fix - now you just need to fix DCO on all commits following the instructions on https://github.com/opensearch-project/project-website/pull/2085/checks?check_run_id=17062276982

**Comment by BigSamu** [comment]
@joshuarrrr Just add the add the signed off all my commits. Forgot to consider that. Many thanks!

**Comment by dtaivpp** [comment]
@BigSamu one more request! Can you move your bio from the `_authors` to the `_authors/osci_2023` directory? Thank you!

**Comment by BigSamu** [comment]
> @BigSamu one more request! Can you move your bio from the `_authors` to the `_authors/osci_2023` directory? Thank you!

HI @dtaivpp, just moved the bio from _authors to _authors/osci_2023 as requested!

**Review by cwperks — APPROVED** [review]
LGTM!

**Comment by BigSamu** [comment]
Hi @dtaivpp @joshuarrrr, quick question. Are the last changes OK to merge? Just want to make sure I am not forgetting something.

Regards,

Samuel

**Review by joshuarrrr — APPROVED** [review]
LGTM

**Pull request merged**